airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -197,6 +198,12 @@ def request_headers(self, **kwargs) -> Mapping[str, Any]:
             headers["Stripe-Account"] = self.account_id
         return headers
 
+    def retry_factor(self) -> float:
```

Comment:
There are two conflicting wishes here:
* Testing the retry mechanism with backoff
* Making sure tests run very fast

I decided to go with the tradeoff of not testing the fact that we sleep during backoff but still testing the retry mechanism so that tests can run fast

Reply:
Yeah that makes sense to me. Since backoff is its own library theoretically we shouldn't have to provide its test coverage.

Reply:
Actually - can we monkeypatch this in conftest instead of wrapping it in production code?